README.md

```diff
@@ -434,6 +467,12 @@ the session is destroyed.
 This optional method is used to delete all sessions from the store. The
 `callback` should be called as `callback(error)` once the store is cleared.
 
+### store.createSession(req, session)
+
+**Required** (available from `express-session`)
+
+This required method is used to create a session from JSON `session` data, set `session.cookie`, and set `req.session`. It should return the new session object. This method will need to utilize `express-session`'s `Cookie` and `Session` objects.
```

Comment:
Does it _need_ to, or would using it simply make it easier to implement? I don't think we do instanceof checks anywhere that makes using them actually a requeirment.